### DISCUSSION TITLE
Changing maxDuration for serverless functions built with SvelteKit doesn't work

### DISCUSSION THREAD

**Discussion by erik-dickens** [vercel/community#5492] [Help]
### Summary

Hey!
In my SvelteKit project I have added configuration according to the [docs](https://vercel.com/docs/functions/configuring-functions/duration) to be able to increase the max duration of serverless function execution. I have a PRO account, but calling the function I still get error:
504: GATEWAY_TIMEOUT
Code: FUNCTION_INVOCATION_TIMEOUT
after 15 s (which is default for PRO accounts)

Any ideas on what could be wrong? (I have PRO account paid plan, so https://github.com/orgs/vercel/discussions/5254 doesn't help)
Thanks!

### Steps to Reproduce

My  svelte.config.js looks like this:
```
import adapter from '@sveltejs/adapter-vercel'
import { vitePreprocess } from '@sveltejs/vite-plugin-svelte'

/** @type {import('@sveltejs/kit').Config} */
const config = {
	// Consult https://kit.svelte.dev/docs/integrations#preprocessors
	// for more information about preprocessors
	preprocess: vitePreprocess(),

	kit: {
		adapter: adapter({
			maxDuration: 30,
		}),
		alias: {
			$css: './src/css',
			$exerciseElements: './src/lib/components/exerciseElements',
			$content: './content',
		},
	},
}

export default config
```
![image](https://github.com/vercel/community/assets/121179746/7ef7c39f-c085-4c13-a9ef-916d6f410515)

![image](https://github.com/vercel/community/assets/121179746/b69d20a9-0681-42ff-80f1-4fb6ca60f3e3)

**Comment by leerob** [comment]
Could you include a public repo with a minimal reproduction of the issue please?

**Reply by erik-dickens** [reply]
Thanks for the attention! Problem is actually solved now. I couldn't reproduce it in a new repo today (the maxDuration worked), and when i tried again in the original repo it also worked. Strange, I don't understand why the problem persisted before. Anyways, thanks

**Comment by erik-dickens** [comment]
Problem is actually solved now. I couldn't reproduce it in a new repo (the maxDuration worked), and when i tried again in the original repo it also worked. Strange, I don't understand why the problem persisted earlier.
